docs/non-normative/how-to-write-conventions/resource-and-entities.md

```diff
@@ -0,0 +1,301 @@
+# Resource and Entities
+
+Table of Contents
+
+- [Modelling Guide](#modelling-guide)
+- FAQ
+  - [When to define a new entity](#when-to-define-a-new-entity)
+  - [When to define a new "is-a" relationship](#when-to-define-an-is-a-relationship-vs-extending-descriptive-attributes)
+  - [How to define identifying attributes](#how-to-define-identifying-attributes)
+  - [How to namespace entities](#how-to-namespace-entities)
+- [Background](#background-resource-and-entities)
+
+## Modelling Guide
+
+To define an entity, create a new semantic convention model file, with a group type as `entity`, for example:
+
+`model/{my_domain}/entities.yaml`:
+
+```yaml
+groups:
+  - id: entity.my_entity
```

Comment:
Is `id` always prefixed by `entity.`? For example for K8s pods is it `entity.k8s.pod` or `k8s.pod`?

Reply:
In semantic conventions, yes. We require this.  The oddities of `id` vs. `name` have been a bit of a thorn in our side in the model.  To keep things reasonably namespaced, semconv requires `{signal}.*` as a prefix for a group denoting a signal.